images/installer/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Using playbook2image as a base
 # See https://github.com/openshift/playbook2image for details on the image
 # including documentation for the settings/env vars referenced below
-FROM registry.centos.org/openshift/playbook2image:latest
+FROM openshift/base-centos7
```

Comment:
Was not sure which one to use but for some reason, `openshift/base-centos7` seemed more inviting. Will switch to just `centos7`